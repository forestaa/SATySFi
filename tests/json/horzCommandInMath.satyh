let-inline \f = {2} in ${\f}
