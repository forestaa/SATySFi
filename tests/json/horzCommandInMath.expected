{
  "file": "horzcommandinmath.satyh",
  "range": {
    "start": { "line": 1, "character": 25 },
    "end": { "line": 1, "character": 27 }
  },
  "message": "! [Type Error] an inline command is used as a math command."
}
